CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.10)
+include(GNUInstallDirs)
```

Comment:
I got a CMake warning for this:
```
Unable to determine default CMAKE_INSTALL_LIBDIR directory because no target architecture is known. Please enable at least one language before including GNUInstallDirs
```

Reply:
Was this on the Raspberry Pi?

Reply:
Yeah it was

Reply:
Fixed now, was to do with the order of statements in the CMake